arches/app/etl_modules/base_data_editor.py

```diff
@@ -51,12 +52,17 @@ def __init__(self, request=None, loadid=None):
         self.node_lookup = {}
 
     def reverse_load(self, loadid):
+        user = (
+            User.objects.get(id=self.userid)
```

Comment:
Seems like we should be storing the `request.user` instead of re-querying

Reply:
But maybe this is not the time to do that if there are too many downstream impacts, it was just an impulse. WIll try to find time to do a real review :-)